README.md

```diff
@@ -1 +1,3 @@
 Windows service template in pure C
+
+Originally written and compiled using MSVS98, this code was updated by Michael Maltsev ([@m417z](https://github.com/m417z)) to allow compilation using newer Visual Studio versions
```

Comment:
> MSVS98

You probably meant MSVC98 :)

Reply:
> You probably meant MSVC98 :)

Thank you, fixed... Ahh been a while since I used Microsoft Stack